<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditEffects</class>
 <widget class="QDialog" name="DialogEditEffects">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Édition des effets</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainFrame.qrc">
    <normaloff>:/MainFrame/Resources/settings.png</normaloff>:/MainFrame/Resources/settings.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="tree">
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkReflect">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Reflet</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkOpacity">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Opacité</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinOpacity">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="check2Sides">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>2 côtés visibles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSelfIlluminate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Pas d'éclairage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkHighlight">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Lampe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="changeTexture">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Changer la texture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CWOrCCW">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>CW &lt;-&gt; CCW</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MainFrame.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogEditEffects</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogEditEffects</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
